docs/components/power-sensor/_index.md

```diff
@@ -0,0 +1,239 @@
+linkTitle: "Power Sensor"
+childTitleEndOverwrite: "Power Sensor"
+weight: 70
+no_list: true
+type: "docs"
+description: "A device that provides information about a robot's systems, including voltage, current, and power consumption."
+tags: ["sensor", "components", "power sensor", "ina219", "ina226", "renogy"]
+icon: "/icons/components/sensor.svg"
+images: ["/icons/components/sensor.svg"]
+# SME: #team-bucket
+---
+
+A power sensor is a device that reports measurements of the voltage, current and power consumption in your robot's system.
+Integrate this component to monitor your power levels.
+
+## Configuration
+
+For configuration information, click on your sensor’s model:
+
+Model | Description <a name="model-table"></a>
+----- | -----------
+[`fake`](./fake/) | a digital power sensor for testing
+[`ina219`](./ina219/) | INA219 power sensor; current and power monitor
+[`ina226`](./ina226/) | INA219 power sensor; current and power monitor
+[`renogy`](./renogy/) | solar charge controller
+
+## Control your power sensor with Viam’s client SDK libraries
+
+To get started using Viam's SDKs to connect to and control your robot, go to your robot's page on [the Viam app](https://app.viam.com), navigate to the **Code sample** tab, select your preferred programming language, and copy the sample code generated.
+
+{{% snippet "show-secret.md" %}}
+
+When executed, this sample code will create a connection to your robot as a client.
+Then control your robot programmatically by adding API method calls as shown in the following examples.
+
+These examples assume you have a power sensor called `"my_power_sensor"` configured as a component of your robot.
+If your power sensor has a different name, change the `name` in the code.
+
+Be sure to import the power sensor package for the SDK you are using:
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+```python
+from viam.components.powersensor import powersensor
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+```go
+import (
+  "go.viam.com/rdk/components/powersensor"
+)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## API
+
+The power sensor component supports the following methods:
+
+{{< readfile "/static/include/components/apis/power-sensor.md" >}}
+
+### GetCurrent
+
+Return the current of a specified device and whether it is AC or DC.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `extra` [(Optional[Dict[str,Any]])](https://docs.python.org/library/typing.html#typing.Optional): Pass additional data and configuration options to the [RPC call](/appendix/glossary/#term-grpc).
+- `timeout` [(Optional[float])](https://docs.python.org/3/library/typing.html#typing.Optional): Specify a time limit (in seconds) for how long `get_current` should wait for a response.
+
+**Returns:**
+
+- [(Tuple[float, bool])](https://docs.python.org/3/library/functions.html#float): A float representing the current reading in amps. A bool indicating whether the current is AC (True) or DC (False).
```

Comment:
This returns the value `true` and `false`, correct? If these are verbatim output from the command, let's render them in monospace with matching case. Let's also specify that the return is a tuple, formed of a float and a bool:

```suggestion
- [(Tuple[float, bool])](https://docs.python.org/3/library/functions.html#float): A tuple which includes a float representing the current reading in amps, and a bool indicating whether the current is AC (`true`) or DC (`false`).
```